gacl/gacl.class.php

```diff
@@ -168,6 +172,14 @@ function __construct($options = NULL) {
 			//Use NUM for slight performance/memory reasons.
 			$this->db->SetFetchMode(ADODB_FETCH_NUM);
 
+            // Set mysql to use ssl, if applicable.
+            // Can support basic encryption by including just the mysql-ca pem (this is mandatory for ssl)
+            // Can also support client based certificate if also include mysql-cert and mysql-key (this is optional for ssl)
+            if (file_exists($GLOBALS['OE_SITE_DIR'] . "/documents/certificates/mysql-ca")) {
+                $this->db->clientFlags = MYSQLI_CLIENT_SSL;
+                error_log("DEBUG: Passing MYSQLI_CLIENT_SSL flag from GACL");
+            }
+
 			// Port to be used in connection
 			$this->db->port = $sqlconf["port"];
 
```

Comment:
At line 186, likely need to change to Connect (ie. remove persistent connection) to support ssl